first_hour_on_vwb/widget_utils.py

```diff
@@ -40,25 +40,24 @@ def list_bq_tables(json_string):
             for table in tables:
                 all_tables.append(f"{row['projectId']}.{row['datasetId']}.{table.table_id}")
     return all_tables
-    
 
-def list_groups(json_string):
+def list_legacy_groups(json_string):
     """ List VWB groups in which user is a member in HTML table form.
     """
     html = f"""<table style='margin: 0 auto,text-align: left'>"""
     json_data = json.loads(json_string)
+    # html += "<th>ORG_ID</th>"
```

Comment:
is this communicating intent of some kind, or a stray comment? there are a few similar commented-out html strings below

Reply:
fixed!